### PR TITLE
Add postgresql support

### PR DESCRIPTION
**Pull request by LoFrance**
Hi,

This PR adds support for **PostgreSQL** and includes **Jest** as the test suite library.

The PostgreSQL library is used in the `delete-list` command to ensure synchronization between Cosmos and PostgreSQL. Due to limitations with the changeFeed set in the `io-developer-portal-service-data` [project](https://github.com/pagopa/io-developer-portal-service-data), we need to handle it separately.

### CODE REVIEW

File: src/utils/postgre.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      port: pipe(
+        getRequiredStringEnv("POSTGRE_DB_PORT"),
+        NumberFromString.decode,
+        E.foldW(() => 5432, identity)
+      ),
+      ssl: true,
+      user: getRequiredStringEnv("POSTGRE_DB_USER"),
+    });
+  }
+  return singletonPool;
```
Comment (AleDore) [inline src/utils/postgre.ts:33]: refactor in a functional way by wrapping `sigletonPool` into an `Option` with `fromNullable`

Comment (LoFrance) [reply src/utils/postgre.ts]: Check it 075832e maybe it's ok now.

File: src/utils/postgre.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      (error) => error as DatabaseError
+    ),
+    TE.mapLeft((e) => {
+      cli.log(chalk.red.bold(`SQL error ${e}`));
+      return e;
+    }),
+    TE.map((q) => {
+      cli.log(chalk.blue.bold(`Subscription deleted from DB`));
+      return q;
+    })
```
[the remark this answers is not in the document]
Comment (LoFrance) [reply src/utils/postgre.ts]: Check it 075832e maybe it's ok now.

**Review by AleDore — APPROVED** [review]
lgtm

**Pull request merged**